controllers/genres.js

```diff
@@ -0,0 +1,38 @@
+const models = require('../models')
+
+const getAllGenres = async (request, response) => {
```

Comment:
This route throws an error

![image](https://user-images.githubusercontent.com/5775716/82481276-14f26c80-9aa3-11ea-8abc-aded51fed4c5.png)